Source/src/modules/ModuleAudio.cpp

```diff
@@ -223,9 +223,20 @@ void Hachiko::ModuleAudio::OptionsMenu()
     {
         SetMusicVolume(volume * 0.1);
     }
+
+    if (ImGui::Button("Reload Sound Banks"))
```

Comment:
We have to take care with this because if you press it and then you play the game it seems that the sound system is not working and if you check it with wwise it doesnt detect the GameObjects.
![image](https://user-images.githubusercontent.com/6739717/187314625-ce1aac7a-731d-4c1e-a5e2-04e99d6a0bd5.png)
But I think it's because it's loading and hasn't finished yet, then it turns to work fine as usual.
Just we have to remember that if we press it we have to wait.

Reply:
Yes @monikamnc , it's right, sorry for not pointing it out. My workaround was play/stop/play and works. It's not perfect but as it's not gonna be widely used I left it as it is to speed up the task. I just needed to avoid closing the engine. Thanks for reviewing!